Jenkinsfile.deploy

```diff
@@ -268,6 +300,7 @@ void commitAndCreatePR(String repo) {
         def prBody = "Generated by build ${BUILD_TAG}: ${BUILD_URL}.\nPlease do not merge, it will be merged automatically after testing."
 
         githubscm.commitChanges(commitMsg, { githubscm.findAndStageNotIgnoredFiles('pom.xml') })
+        githubscm.commitChanges(commitMsg, { githubscm.findAndStageNotIgnoredFiles('build.gradle') })
```

Comment:
That means I need to change a closure in findAndStageNotIgnoredFiles and there is two options I can pass
Use file names: `find -name "pom.xml" -or -name "build.gradle" `
Use regexp: find . -regex '.*\(pom\.xml\|build\.gradle\)'
WYT fits best?

Reply:
The `findAndStageNotIgnoredFiles` already uses `find` inside; how about:
```suggestion
        githubscm.commitChanges(commitMsg, { 
            githubscm.findAndStageNotIgnoredFiles('pom.xml') 
            githubscm.findAndStageNotIgnoredFiles('build.gradle')
        })
```

Reply:
Sure exactly. I just thought that we can change it so we do not to stage every time when we want to commit a new file. Since there is only two of them it should be fine.

Reply:
There is an issue occurred  when git add [empty] returns exit code 1
See: https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/129